fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoanTransactionsApiResourceSwagger.java

```diff
@@ -275,6 +276,12 @@ private PaymentType() {}
         }
     }
 
+    @Schema(description = "GetLoansLoanIdTransactionsResponse")
+    public abstract static class GetLoansLoanIdTransactionsResponse implements Page<GetLoansLoanIdTransactionsTransactionIdResponse> {
```

Comment:
Is there a reason why is this abstract?

Reply:
I didn't want to fully implement the Page interface for it, and it's here because swagger can't take response spec with type parameter in it.